manim/mobject/types/vectorized_mobject.py

```diff
@@ -442,17 +449,17 @@ def add_cubic_bezier_curve_to(self, handle1, handle2, anchor):
         else:
             self.append_points([self.get_last_point()] + new_points)
 
-    def add_line_to(self, point):
+    def add_line_to(self, point: Vector):
         nppcc = self.n_points_per_cubic_curve
         self.add_cubic_bezier_curve_to(*[
             interpolate(self.get_last_point(), point, a)
             for a in np.linspace(0, 1, nppcc)[1:]
         ])
         return self
 
-    def add_smooth_curve_to(self, *points):
+    def add_smooth_curve_to(self, *points: List[Vector]):
```

Comment:
*points: Vector   (see [this](https://stackoverflow.com/questions/37031928/type-annotations-for-args-and-kwargs))
(also varargs are passed as a tuple not as a list, but anyway)
```suggestion
    def add_smooth_curve_to(self, *points: Vector):
```